create-pull-request.js

```diff
@@ -11,12 +11,27 @@ const open = require("open");
       throw stderr;
     }
 
+    const browsers = [
+      { id: "chrome", value: "chrome" },
+      { id: "firefox", value: "firefox" },
+      { id: "edge", value: "msedge" },
+      { id: "safari", value: "safari" },
+      { id: "ie", value: "iexplore" },
+      { id: "opera", value: "opera" }
+    ];
+
     const branchName = stdout.trim();
     const remoteOriginUrl = await gitRemoteOriginUrl();
     const newPullRequestUrl = `https://${remoteOriginUrl.replace("git@", "").replace(":", "/").replace(/\.git$/, "")}/pull/new/${branchName}`;
+    let browser = process.argv.pop();
```

Comment:
I don't think you should be doing this because if I add more params this will get the wrong param.
```bash
$ create-pull-request firefox hello
```
In this case, it would get `hello`

```suggestion
    let [, browser = '']= process.argv;
```